ci/install.sh

```diff
@@ -11,13 +11,13 @@ main() {
         target=x86_64-apple-darwin
     fi
 
-    # TODO At some point you'll probably want to use a newer release of `cross`,
-    # simply change the argument to `--tag`.
+    local tag="$(git ls-remote --tags --refs --exit-code https://github.com/japaric/cross | cut -d/ -f3 | sort --version-sort | tail -n1)"
```

Comment:
Let's filter out the `-rc` versions and stick to version 0.1.x to prevent breaking changes (see the grep command I gave you before). Sometimes `-rc` versions may show up as the latest release when I'm testing deploys.